CONTRIBUTING.md

```diff
@@ -323,7 +332,64 @@ Update the version's release date and status from the
 [Manage Versions](https://jira.mongodb.org/plugins/servlet/project-config/PHPC/versions)
 page.
 
-### Update version info
+### Trigger the release workflow
+
+Releases are done automatically through a GitHub Action. Visit the corresponding
+[Release New Version](https://github.com/mongodb/mongo-php-driver/actions/workflows/release.yml)
+workflow page to trigger a new build. Select the correct branch (e.g. `v1.18`)
+and trigger a new run using the "Run workflow" button. In the following prompt,
+enter the version number and the corresponding JIRA release number for the 
+release. This will create and push the necessary commits and tag, create a draft
+release, and trigger the packaging builds for the newly created tag. The release
+is created in a draft state and can be published once the release notes have 
+been updated.
+
+### Upload package to PECL
+
+Once the packaging workflow has finished creating the PECL package, it will be
+published as a build artifact of the package workflow, as well as a release
+artifact in the draft release. This package may be published via the
+[Release Upload](https://pecl.php.net/release-upload.php) form. You will have
+one chance to confirm the package information after uploading.
+
+> **Note:** If downloading the package from the build artifacts, be aware that
+> these are always provided as zip files. You'll have to unpack the tgz archive
+> prior to uploading it to PECL.
```

Comment:
Is this a quirk of [gh release upload](https://cli.github.com/manual/gh_release_upload) that everything gets wrapped in a zip file? I don't see that mentioned in the GitHub docs.

Reply:
Release assets (so what we upload using `gh release upload` are left as-is - but build artifacts are always zip files, even if we only upload a single file. Quoting the [documentation](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts):
> When an artifact is downloaded using the GitHub UI, all files that were individually uploaded as part of the artifact get zipped together into a single file.